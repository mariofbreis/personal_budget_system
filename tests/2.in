i 100 0 0 r Receita
i 200 1 0 r Receita
i 300 1 0 r Receita
i 400 0 0 r Receita
i 50 999 0 d Despesa
i 51 999 0 d Despesa
i 52 999 0 d Despesa
i 53 999 0 d Despesa
i 50 500 0 r Receita
i 50 500 0 d Despesa
l 0
l 1
l 999
l 500
l 0
