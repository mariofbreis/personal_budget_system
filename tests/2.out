0 100 r Receita
1 400 r Receita
0 200 r Receita
1 300 r Receita
0 50 d Despesa
1 51 d Despesa
2 52 d Despesa
3 53 d Despesa
0 50 r Receita
1 50 d Despesa
0 100 r Receita
1 400 r Receita
